vbfml/scripts/err_analysis

```diff
@@ -443,10 +456,28 @@ def evaluate(ctx, input_dir: str, sequence_type: str) -> None:
     default="history",
     help="If specified, clarify the name of the pdf save file",
 )
-def compare_hist(ctx, input_dirs: str, number_epochs: int, save_name: str) -> None:
+def compare_hist(
+    ctx, input_dirs: str, number_epochs: int, save_name: str, is_parent: bool = False
+) -> None:
     """
     Compare the statistics of the history of different models.
     """
+    if is_parent:
+        assert (
+            len(input_dirs) == 1
+        ), f"too many '-i' argument, if '--is-parent' is flaged, only one '-i' is accepted"
+        input_dirs = glob(pjoin(input_dirs[0], f"*"))
+
+        remove_list = []
+        for dir in input_dirs:
+            if not os.path.isdir(dir):
+                remove_list.append(dir)
+            else:
+                if not os.path.exists(pjoin(dir, "history.pkl")):
+                    remove_list.append(dir)
+
+        for el in remove_list:
+            input_dirs.remove(el)
```

Comment:
Thanks for adding this check, I think there might be an easier way to do this. When you loop over `input_dirs`, can't we just `continue`, if one of our checks fail, like this:

```python
for dir in input_dirs:
    # Check if dir is a valid directory that contains history.pkl file
    if not os.path.isdir(dir) or not os.path.exists(pjoin(dir, "history.pkl")):
        continue
    # Then do the processing here
```

I think this would be easier + cleaner then updating the `input_dirs` beforehand

Reply:
done